src/dialogs/config-flow/step-flow-form.ts

```diff
@@ -73,25 +64,13 @@ class StepFlowForm extends LitElement {
             `
           : html`
               <div>
-                <mwc-button
-                  @click=${this._submitStep}
-                  .disabled=${!allRequiredInfoFilledIn}
```

Comment:
Since we now use the `input` event instead of `change` event, we could revert this change? Or do you like the new UX better?

Reply:
I think I like it more to not disable it.